Update _index.es.md

AlexAndradeNet opened the pull request:
Translation to Spanish.

I've agreed the CLA

AlexAndradeNet committed:
Update _index.es.md

Translation to Spanish.

I've agreed the CLA

diemol commented on docs_source_files/content/grid/_index.es.md:
```diff
@@ -4,30 +4,14 @@ chapter: true
-(instead of in each individual test).
+Selenium Grid nos permite ejecutar pruebas en paralelo en múltiples máquinas, y administrar diferentes versiones y configuraciones de navegador de forma centralizada (en lugar de en cada prueba individual).
+
+Selenium Grid no es una bala de plata. Resuelve un subconjunto de problemas comunes de delegación y distribución, pero, por ejemplo, no administrará su infraestructura y podría no satisfacer sus necesidades específicas.
```
`bala de plata` is really a Spanish expression? Could you please replace it with something more understandable?

AlexAndradeNet replied:
The "bala de plata" (silver bullet) expression was changed for a more comprehensible frase

AlexAndradeNet committed:
Update _index.es.md

The "bala de plata" (silver bullet) expression was changed for a more comprehensible frase

AlexAndradeNet reviewed (commented):
The "bala de plata" (silver bullet) expression was changed for a more comprehensible frase

diemol reviewed (approved):
Thank you @AlexAndradeSan!

diemol committed:
Merge branch 'gh-pages' into patch-23
